docs/checks/internal/checks.yaml

```diff
@@ -139,12 +139,6 @@ checks:
       status checks before acceptance into a main branch, or preventing rewriting of
       public history.
 
-      Note: The following settings queried by the Branch-Protection check require an admin token: `DismissStaleReviews`, `EnforceAdmin`, and `StrictStatusCheck`. If
```

Comment:
Should we leave this as is since the status-quo does not change with this PR?

Reply:
absolutely. I forgot to put it back in.